frontend/src/lib/components/BillingAlerts.tsx

```diff
@@ -20,9 +20,8 @@ export function BillingAlerts(): JSX.Element | null {
         isWarning = true
         message = (
             <p>
-                <b>Warning!</b> You have already used{' '}
-                <b className="text-warning">{freePlanPercentage && freePlanPercentage * 100}%</b> of your 1 million free
-                events this month.{' '}
+                <b>Warning!</b> You have already used <b className="text-warning">{percentage && percentage * 100}%</b>{' '}
```

Comment:
if `percentage` is falsey then we'll say "You have already used % of your 1 million free events this month"

Should we wrap the entire sentence in the `percentage &&` check to remove that (probably unlikely) possibility

Reply:
Done!